index.html

```diff
@@ -10,156 +10,183 @@
 </head>
 <body>
   <header class="navbar">
-  <nav class="container">
-    <a  class="logo" href="#">My logo</a>
-    <img class="mobile-menu" src="img/Union.png" alt="open navigation">
-  </nav>
-</header>
-<section class="hero">
+    <div class="nav-logo">
+      <a  class="logo" href="#">My logo</a>
+    </div>
+    <nav class="container">
+      <div class="menu-wrapper">
+        <img class="mobile-menu" src="img/Union.png" alt="open navigation">
+      </div>
+      <ul class="nav-list">
+        <li class="nav-item"><a href="#portfolio">Portfolio</a></li>
+        <li class="nav-item"><a href="#about">about</a></li>
+        <li class="nav-item"><a href="#contact">contact</a></li>
+      </ul>
+    </nav>
+  </header>
+<section id="portfolio" class="hero">
   <div class="container">
-  <h2 class="primary-text">I’m Asiati Magana <br> Glade to see you!</h2>
-  </div>
-  <p class="secondary-text"> 
-    I’m a software developer! I can help you build a product ,
-    feature or website Look through some of my work and experience!
-     If you like what you see and have a project you need coded,
-      don’t hestiate to contact me.
+    <h2 class="primary-text">I’m Asiati Magana <br> Glade to see you!</h2>
+    <p class="secondary-text"> 
+      I’m a software developer! I can help you build a product ,
+      feature or website Look through some of my work and experience!
+      If you like what you see and have a project you need coded,
+      don’t hesitate to contact me.
```

Comment:
- [ ] Good job making some of the required changes, just use a bigger size for all the texts in the hero section to make it look more similar to the Figma design as the last reviewer said:

Template:
![Captura de pantalla 2022-03-25 a la(s) 8 47 08 p  m](https://user-images.githubusercontent.com/83717285/160220143-015b1d5b-9523-4024-91f6-b28ca761c0ac.png)

Your design:
![Captura de pantalla 2022-03-25 a la(s) 8 46 14 p  m](https://user-images.githubusercontent.com/83717285/160220164-a20a8dba-dd4d-48cd-a94e-356f3ff08a30.png)